### PR TITLE
Created Tabular and Stacked inline.

### PR DESCRIPTION
**Pull request by JostCrow**
Now the user can choose between stacked and tabular inline.

### CODE REVIEW

File: ordered_model/admin.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def _get_model_info(self):
         }
 
 
-class OrderedTabularInline(admin.TabularInline):
+class OrderedInlineMixin:
```
Comment (bfirsh) [inline ordered_model/admin.py]: Should be `class OrderedInlineMixin(object):`, I think...

Comment (JostCrow) [reply ordered_model/admin.py]: Yes you are correct. I will change it.

**Comment by JostCrow** [comment]
I have updated the documentation

**Comment by JostCrow** [comment]
@bfirsh

**Comment by bfirsh** [comment]
Ah, thanks for the ping! Let me take a look.

**Comment by bfirsh** [comment]
Looks good! Thank you! ✨

**Pull request merged**